lib/async/scheduler.rb

```diff
@@ -291,6 +291,16 @@ def process_wait(pid, flags)
 			return @selector.process_wait(Fiber.current, pid, flags)
 		end
 		
+		def blocking_operation_wait(work)
```

Comment:
What if this section had an API documentation note, telling the reader what Ruby version this exists in, etc?

```suggestion
        # Implements Fiber::Scheduler#blocking_operation_wait support. This exists in Ruby 3.3.x.
		def blocking_operation_wait(work)
```

Reply:
My version number there is just a placeholder for the real thing.

Reply:
Ah yes, this should land in 3.4. It's probably a good idea to document more of the scheduler.